FSQCellManifest/FSQCellManifest.h

```diff
@@ -449,32 +450,32 @@ extern const NSInteger kRowIndexForFooterIndexPaths;
  
  @return A new table view cell manifest.
  */
-- (instancetype)initWithDelegate:(id)delegate 
-                         plugins:(NSArray *)plugins
+- (instancetype)initWithDelegate:(nullable id)delegate
```

Comment:
Is there a reason we shouldn't make the DelegateType a generic parameter of the manifest?